my-app/src/App.css

```diff
@@ -0,0 +1,112 @@
+* {
+  font-family: "Lato", sans-serif;
+  margin: 0;
+  padding: 0;
+  /* font-size: 12px; */
+}
+
+body,
+html {
+  margin: 0;
+  padding: 0;
+}
+
+.post {
+  background: #ffffff;
+  font-family: "Lato", sans-serif;
+  padding: 8px 8px;
+  margin: 0 24px;
+  color: #37474f;
```

Comment:
This color seems to be re-used in a few other places (together with `#0277bd`) which makes this hard to maintain.

There are two ways you could make this a bit cleaner:
1. Use CSS variables - they're not supported in IE11, but for this project that would be ok
2. Define a separate class (i.e. `.dark-blue`) which only holds the color rule. Then, add that class to the elements that need to be this color.

Reply:
Agreed, went ahead and added css variables: --light-blue and --dark-blue.